infra/lib/ci-cluster.ts

```diff
@@ -26,7 +25,7 @@ export class CICluster extends cdk.Construct {
     super(scope, id);
 
     this.testCluster = new eks.Cluster(scope, 'TestInfraCluster', {
-      version: eks.KubernetesVersion.V1_19,
+      version: eks.KubernetesVersion.V1_20,
```

Comment:
was this necessary for the upgrade to Flux or were you just using this PR as an opportunity to bring our EKS cluster up to date?

Reply:
It was necessary. Flux installation fails with a pre-check if the cluster is below k8s 1.20.6